src/EnergyPlus/DXCoils.cc

```diff
@@ -10407,9 +10409,12 @@ Label50: ;
 		Real64 OutletAirHumRat; // outlet air humidity ratio [kg/kg]
 		Real64 TotCapHS; // total capacity at high speed [W]
```

Comment:
This equation looks correct to me:
DXCoil( DXCoilNum ).ElecHeatingPower = TotCap * EIR * DXCoil( DXCoilNum ).HeatingCoilRuntimeFraction * InputPowerMultiplier;

TotCap * EIR is the power wo defrost, and TotCap * EIR * InputPowerMultiplier is the power with defrost.

I would add that the zone load does not change due to "resistive" defrost and the heating capacity needed to meet that load, purely for heating sake, should not change. Reverse cycle defrost is very different.

Lastly, COP is externally calculated so nothing else to review here.